apps/oc_chef_authz/Makefile

```diff
@@ -1,7 +1,6 @@
 DEPS = $(CURDIR)/deps
 
-# TODO automate managing this revision
-EC_SCHEMA_VERSION = 2.8.0
+EC_SCHEMA_VERSION?=master
```

Comment:
The very next thing we are planning to do is pull esc-schema into oc_erchef, so its probably fine to leave at master.